spec/fixtures/application/1.0/application.json

```diff
@@ -38,7 +38,31 @@
       },
       "correspondence_address": null,
       "residence_type": null,
-      "relationship_to_owner_of_usual_home_address": null
+      "relationship_to_owner_of_usual_home_address": null,
+      "has_partner": "yes",
+      "relationship_to_partner": "living_together",
+      "separation_date": "2023-06-09",
+      "relationship_status": null
+    },
+    "partner": {
+      "first_name": "Jennifer",
+      "last_name": "Holland",
+      "other_names": "Diane",
+      "nino": "AB123456C",
+      "benefit_type": null,
+      "last_jsa_appointment_date": null,
+      "date_of_birth": "2001-12-23",
+      "home_address": {
+        "lookup_id": null,
+        "address_line_one": "1 Road",
+        "address_line_two": "Village",
+        "city": "Some nice city",
+        "country": "United Kingdom",
+        "postcode": "SW1A 2AA"
+      },
+      "involvement_in_case": "codefendant",
+      "conflict_of_interest": "no",
+      "has_same_address_as_client": "yes"
```

Comment:
Worth putting this as `no` to make it consistent with having a home address recorded on partner?

Reply:
I removed the home address on partner, but maybe there is an argument to have more fixtures? we could have one for unemployed partner and one for unemployed no partner.